tshirts.c

```diff
@@ -6,9 +6,9 @@ char size(int cms) {
     char sizeName = '\0';
     if(cms < 38) {
         sizeName = 'S';
-    } else if(cms > 38 && cms < 42) {
+    } else if(cms >= 38 && cms < 42) {

```

Comment:
The check on `>= 38` isn't required because the condition is already checked once and you have used an `else`.
Keeping the check at one place also means we need to change only in one place.